plot_fits.py

```diff
@@ -291,12 +291,19 @@ def _parser():
                         'or both.')
     parser.add_argument('--ftype', help='Select which type the fits file is',
                         choices=['ARES', 'CRIRES'], default='ARES')
+    parser.add_argument('--fitsext', help='Select Crires extention',
+                        choices=['0', '1', '2', '3', '4'], default='0')
     args = parser.parse_args()
     return args
 
 
 def main(fname, lines=False, model=False, telluric=False, sun=False,
-         rv=False, rv1=False, rv2=False, ccf='none', ftype='ARES'):
+         rv=False, rv1=False, rv2=False, ccf='none', ftype='ARES', 
+<<<<<<< HEAD
+         fitsext='0'):
+=======
+         fitsext='none'):
+>>>>>>> fe8f32765887481754cd38bd686c1e7e24deb0ab
```

Comment:
It looks like you have a merge conflict here. Please select which option you prefer.